services/ui-src/src/components/export/ExportedModalOverlayReportSection.tsx

```diff
@@ -149,14 +149,15 @@ export function renderModalOverlayTableBody(
   switch (reportType) {
     case ReportType.WP:
       return entities.map((entity, idx) => {
+        // console.log("entity", entity);
         return (
           <Box sx={sx.container}>
             <Tr key={idx}>
               <Td sx={sx.statusIcon}>
                 <EntityStatusIcon
                   entity={entity}
                   isPdf={true}
-                  entityStatus={getInitiativeStatus(report, entity)}
+                  entityStatus={getInitiativeStatus(report, entity, true)}
```

Comment:
suggestion: I see on the line above we also have `isPdf={true}`. Could we maybe set this variable and use the variable instead so we know what the `true` represents in the call to `getInitiativeStatus`?